src/Pos/Types/Block/Functions.hs

```diff
@@ -183,16 +186,20 @@ recreateMainBlock _gbHeader _gbBody _gbExtra = do
         Left err -> fail $ toString err
     pure gb
 
+-- | Dummy implementation of make genesis extra header data.
+mkGenesisEHD :: () -> GenesisExtraHeaderData
+mkGenesisEHD = GenesisExtraHeaderData . mkAttributes
```

Comment:
Which attributes? There will be no attributes. If later we decide to add something as _attributes_, we will change type of header. For example, if we want to add another proof to genesis header, we will put it into `GenesisProof`. We will deserialize it from attributes and serialize it as attributes. But it will be stored in `GenesisProof`.